docs/install/1-install-server.rst

```diff
@@ -189,8 +189,9 @@ Install the software
     Create a repositories file to point yum to the inmanta service orchestrator release repository. Create a file
     ``/etc/yum.repos.d/inmanta.repo`` with the following content:
 
+
     .. code-block:: sh
-       :substitutions:
+        :substitutions:
```

Comment:
This was causing the whole block to be rendered with a leading space. That lead to some error when copy pasting this block into a config file